src/transport/SecureSessionMgr.h

```diff
@@ -44,6 +44,49 @@ namespace chip {
 
 class SecureSessionMgr;
 
+/**
+ * @brief
+ *  Tracks ownership of a encrypted PacketBuffer.
+ *
+ *  EncryptedPacketBufferHandle is a kind of PacketBufferHandle class and used to hold a PacketBuffer
+ *  object whose payload has already been encrypted.
+ */
+class EncryptedPacketBufferHandle : public System::PacketBufferHandle
```

Comment:
Let's mark this class as `final`, so that it cannot be extended.

Reply:
Acked